src/Enum.php

```diff
@@ -86,6 +86,7 @@ public function getKey()
 
     /**
      * @psalm-pure
+     * @psalm-suppress InvalidCast
```

Comment:
This change doesn't seem to match the title of the commit.

Reply:
Oh yes, somehow I messed these commit messages up! Fix incoming